app/page.tsx

```diff
@@ -251,6 +342,44 @@ export default function HomePage() {
               <Icon name="chevron-down" />
             </Button>
           )}
+
+          {showClientUI ? (
+            <Popover
+              open={isStatePopoverOpen}
+              onOpenChange={setStatePopoverOpen}
+            >
+              <PopoverTrigger>
```

Comment:
I'd place this `asChild` prop in all the `PopoverTrigger`components in this page, so when we publish the updates on Button components, the error we receive in the console will disappear without new changes
```suggestion
              <PopoverTrigger asChild>
```

Reply:
That's right, thanks!